pymdownx/superfences.py

```diff
@@ -46,22 +46,18 @@
 
 RE_NESTED_FENCE_START = re.compile(
     r'''(?x)
-    (?P<fence>~{3,}|`{3,})[ \t]*                                              # Fence opening
-    (\{?                                                                      # Language opening
-    \.?(?P<lang>[\w#.+-]*))?[ \t]*                                            # Language
-    (?:
-    hl_lines=(?P<quot>"|')(?P<hl_lines>\d+(?:[ \t]+\d+)*)(?P=quot)[ \t]*|     # highlight lines
-    linenums=(?P<quot2>"|')                                                   # Line numbers
-        (?P<linestart>[\d]+)                                                  #   Line number start
-        (?:[ \t]+(?P<linestep>[\d]+))?                                        #   Line step
-        (?:[ \t]+(?P<linespecial>[\d]+))?                                     #   Line special
-    (?P=quot2)[ \t]*|
-    (?P<tab>tab=)(?:(?P<quot3>"|')(?P<tab_title>.*?)(?P=quot3))?[ \t]*        # Tab specifier
-    )*
-    }?[ \t]*$                                                                 # Language closing
+    (?P<fence>~{3,}|`{3,})[ \t]*                                                          # Fence opening
+    (\{?                                                                                  # Language opening
+    \.?(?P<lang>[\w#.+-]*))?[ \t]*                                                        # Language
+    (?P<options>(?:\b[a-zA-Z][a-zA-Z0-9_]*=(?:(?P<quot>"|')[^"'\n\r]*(?P=quot))?[ \t]*)*) # Options
```

Comment:
This is evaluated line by line, so it is probably not necessary to exclude `\n\r`....not sure what I was thinking here.